model-optimizer/extensions/front/RollWithEmptyAxesReplacer.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.front.AttributedRollToRoll import AttributedRollToRoll
+from extensions.ops.roll import Roll
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph, rename_nodes
+from mo.ops.const import Const
+from mo.ops.reshape import Reshape
+from mo.ops.shape import Shape
+
+
+class RollWithEmptyAxesReplacer(FrontReplacementOp):
+    """
+    This transformation replaces Roll with empty axes input with the following sequence of operations:
```

Comment:
Why we need such transformation?

Reply:
Axes is optional parameter for MxNet. If it is not specified input tensor should be flattened, then roll is applied on "0" axis, then tensor is reshaped to original shape. During specification discussion it was decided to implement this mode as a transformation in MO.

Reply:
Please, update the comments in the code describing why the transformation works that way. I mean that you can refer to MxNet operation semantic.